src/editorAdapter.ts

```diff
@@ -68,6 +95,11 @@ export default function adapter(data: OPDSEntry): BookData {
   for (const author of data.authors) {
     authors.push(Object.assign({}, author, { role: "aut" }));
   }
+  const contributors = data.contributors?.map((contributor) => ({
+    name: contributor.name,
```

Comment:
Should be `contributor?.`, as in the lines below?

Reply:
Fixed.